pkg/util/admissionfairsharing/admission_fair_sharing.go

```diff
@@ -45,6 +45,11 @@ func CalculateAlphaRate(sampling, halfLifeTime float64) float64 {
 }
 
 func CalculateEntryPenalty(totalRequests corev1.ResourceList, afs *config.AdmissionFairSharing) corev1.ResourceList {
+	// Return empty penalty if no admission fair sharing config is provided
+	if afs == nil {
+		return corev1.ResourceList{}
+	}
+
```

Comment:
Should we disable entry penalty calculation at the scheduler level?

https://github.com/kubernetes-sigs/kueue/blob/ed155a674d27fd47c8d2d4a790ee920110ffa91b/pkg/scheduler/scheduler.go#L865

IIUC, when AFS is disabled, we can avoid entry penalty, right?